source/draw/DrawContext.ooc

```diff
@@ -66,7 +68,7 @@ DrawContext: abstract class {
 		IntVector2D new(width, height)
 	}
 	getTextPixelBounds: static func (message: Text, fontAtlas: Image) -> IntVector2D {
-		tileBounds := This getTextIndexBounds(message)
+		tileBounds := This getTextIndexBounds(message take())
```

Comment:
If you just want to pass the message to another function, you should be ok without `take() / free() calls`, like
```
getTextPixelBounds: static func (message: Text, fontAtlas: Image) -> IntVector2D {
		tileBounds := This getTextIndexBounds(message)
		tileBounds * This getFontSize(fontAtlas)
}
```